lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -966,6 +980,9 @@ def get_proj(self):
 
         dist is the distance of the eye viewing point from the object point.
         """
+        # chosen for similarity with the initial view before gh-8896
+        pb_aspect = np.array([4, 4, 3]) / 3.5
```

Comment:
I imagine that there are going to be some requests for an API to change this, for people who liked their 1:1:1 plot boxes. I started working on that two years ago, but it opened a can of worms that I didn't want to open concurrently with this PR. At some point I'll see if I can rebase those changes.